packages/cadl-rlc-test/test/authoring/spec/main.cadl

```diff
@@ -100,6 +102,12 @@ model Deployment {
   name: string;
 }
 
+@doc("Deployment resource creation parameters.")
+@withVisibility("create")
+model DeploymentCreationParameters {
+  ...Deployment;
+}
```

Comment:
I was wondering if we could add this part into testing not in smoke-test.

Because we sync the authoring cadl from cadl project. If we add here we would manully maintain the effort.

Reply:
Because this needs additional decorator not sure how to do that in the unit test

Reply:
Could we keep it to a sperate RP?